src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -138,9 +133,6 @@ public Response createDataAccessRequestWithDAARestrictions(
       @Auth AuthUser authUser, @Context UriInfo info, String dar) {
     try {
       User user = findUserByEmail(authUser.getEmail());
-      if (user.getLibraryCards().isEmpty()) {
```

Comment:
This looks good as well.  It's wrapped, and also calling createDataAccessRequest.